doc/changelog/02-changed/1333-seminorm.md

```diff
@@ -0,0 +1,8 @@
+- in `ssrnum.v`
+  + Mixin `Zmodule_isNormed` is now a factory building a
+    `SemiNormedZmodule` and a `SemiNormedZmodule_isPositiveDefinite`.
+    All lemmas from the theory of `normedZmodType` were generalized
+    to `semiNormedZmodType` but for the ones that require a
+    `normedZmodType`, namely `normr0_eq0`, `normr0P` and `normr_eq0`.
```

Comment:
The negative form is weird and not find-friendly, and it costs basically nothing to list the lemmas that you did generalize.

Reply:
agreed